source/frontend/src/features/properties/map/acquisition/update/stakeholders/UpdateStakeHolderYupSchema.ts

```diff
@@ -0,0 +1,17 @@
+/* eslint-disable no-template-curly-in-string */
+import * as Yup from 'yup';
+
+export const UpdateStakeHolderYupSchema = Yup.object().shape({
+  interestHolders: Yup.array().of(
+    Yup.object().shape({
+      legalSurveyPlanNum: Yup.string().max(
+        250,
+        'Legal survey plan must be less than 250 characters',
+      ),
+      interestTypeCode: Yup.string().required('Interest type is required').nullable(),
+      impactedProperties: Yup.array().min(1, 'At lease one impacted property is required'),
+      contact: Yup.object().required('Interest holder is required').nullable(),
+    }),
+  ),
+  nonInterestPayees: Yup.array().min(0),
```

Comment:
I'm a bit confused about this form validation schema: 

- I couldn't find the field "legalSurveyPlanNum" in the update stakeholder form
- Not sure what making a field required and then nullable would accomplish...
  - e.g. `Yup.string().required('Interest type is required').nullable(),`
- I would assume "nonInterestPayees" would validate "impactedProperties" (in the same manner it gets validated for "interestHolders"). I couldn't find in Confluence anything saying nonInterestPayees could be added without impacted properties

Reply:
thanks, corrected.